src/types/port-rpc-events.d.ts

```diff
@@ -26,17 +26,15 @@ export type GuestToHostEvent =
   | 'highlightsVisibleChanged';
 
 /**
- * Events that the guest sends to the sidebar
+ * Events that guests send to the sidebar.
  */
 export type GuestToSidebarEvent =
```

Comment:
We probably need to revisit the values here, as these are both events and commands, but they are inconsistently spelled as commands or events.

* `closeSidebar`, `openSidebar` -> These are in fact commands where guests request an action to be performed.
* `createAnnotation` -> This is an event, so I guess it should be renamed as `annotationCreated`?
* `documentInfoChanged` -> This is in fact an event, as the name suggests.